integration-tests/go.mod

```diff
@@ -8,20 +8,20 @@ require (
 	github.com/rs/zerolog v1.30.0
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
 	github.com/smartcontractkit/caigo v0.0.0-20230621050857-b29a4ca8c704
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20231117021201-6814387d8d3e
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20231123003013-379db0b9e4c7
 	github.com/smartcontractkit/chainlink-starknet/ops v0.0.0-20230329050701-40e3b18bb026
-	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.1-beta-test.0.20231116220530-9d5db8010d3b
+	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.1-beta-test.0.20231117204155-b253a2f56664
 	github.com/smartcontractkit/chainlink/integration-tests v0.0.0-20231116224343-7780eb63d8dc
-	github.com/smartcontractkit/chainlink/v2 v2.5.1-0.20231116224343-7780eb63d8dc
+	github.com/smartcontractkit/chainlink/v2 v2.5.1-0.20231122111232-f7c254bc9c73
```

Comment:
We typically keep these two aligned with the same hash, since integration-tests uses a relative reference to chainlink/v2.
Let's update `integration-tests` to f7c254bc9c73